models/members.js

```diff
@@ -5,21 +5,24 @@
 
 const firestore = require("../utils/firestore");
 const userModel = firestore.collection("users");
-
+const { ROLES } = require("../constants/roles");
 /**
  * Fetches the data about our users
  * @return {Promise<userModel|Array>}
  */
 
-const fetchUsers = async () => {
+const fetchUsers = async (queryParams = {}) => {
   try {
     const snapshot = await userModel.get();
-
     const allMembers = [];
 
+    let { showArchived } = queryParams;
+    showArchived = showArchived === "true";
```

Comment:
We are converting a string variable to boolean here? Can we use a new variable `isArchivedShown`?

Reply:
Automatic types conversion is how we have bugs later.

Reply:
We could have used a new variable here. 
But since the only values showArchived can have are 'true' or 'false' and it's intended to be used as a boolean further in the code so I guess it won't be a problem